CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.0.10] - 2017-10-05
+### Fixed
+- The oauth redirect URL is no trailing-backslash insensitive
```

Comment:
I think this is a tiny bit hard to parse with the double negative
it's no longer sensitive to a trailing-backslash right?

Reply:
woops left out the "longer"